### PR TITLE
Test Mosaic Tutorial Post 2.11

### PR DESCRIPTION
**Pull request by sekyondaMeta**
Test mosaic tutorial post 2.11 release

Disabled GPT-2 dropout (resid_pdrop=0, attn_pdrop=0, embd_pdrop=0) in run_training_ac() to work around a PyTorch 2.11 bug where the CUDA dropout kernel crashes during gradient checkpointing recomputation (#3774). Dropout has no impact on this tutorial's purpose of memory profiling with Mosaic.

cc @basilwong

### CODE REVIEW

**Comment by sekyondaMeta** [comment]
@basilwong here is an attempt to get this to work. It is a bandaid at best, still needs an actual fix
